iceoryx_posh/source/roudi/roudi_cmd_line_parser.cpp

```diff
@@ -159,7 +159,7 @@ CmdLineParser::parse(int argc, char* argv[], const CmdLineArgumentParsingMode cm
             else
             {
                 m_processKillDelay =
-                    units::Duration::seconds(static_cast<unsigned long long int>(processKillDelayInSeconds));
+                    units::Duration::fromSeconds(static_cast<unsigned long long int>(processKillDelayInSeconds));
```

Comment:
same here, please remove the cast. It an oversight from the last refactoring.

Reply:
Done.